python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -42,7 +46,7 @@ class MineResource(Resource, UserMixin, ErrorMixin):
             'mine_no_or_guid':
             'Mine number or guid. If not provided a paginated list of mines will be returned.'
         })
-    @requires_role_mine_view
+    # @requires_role_mine_view
```

Comment:
Should this comment be removed or is this truly public?

Reply:
good catch